src/utils/openmc/statepoint.py

```diff
@@ -7,6 +8,9 @@
 import openmc
 from openmc.constants import *
 
+if sys.version > '3':
```

Comment:
Perhaps we should make Python 3 the standard for Travis? Although I use Python 3 from time-to-time, I still haven't made it the default on my systems, hence issues like this arise. Anyway, just a thought.